gluon-mesh-vpn-wireguard/files/lib/gluon/gluon-mesh-wireguard/checkuplink

```diff
@@ -22,25 +29,25 @@ if [ "$(uci get wireguard.wireguard.enabled)" == "true" ] || [ "$(uci get wiregu
                 CONNECTED=0
         fi
 
-        # Gretapinterface vorhanden??
+        # VXLAN Interface vorhanden??
         if [ $CONNECTED != 0 ]; then
-                ip addr show dev gre > /dev/null
+                ip addr show dev vxlan-wan > /dev/null
```

Comment:
What kind of interface are you expecting here?

Reply:
I am not sure if this check really makes sense it seems to check if the vxlan is there ... and deletes everything if not.